api/python/cellxgene_census/tests/experimental/ml/test_geneformer.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import tiledbsoma
+
+import cellxgene_census
+
+try:
+    from cellxgene_census.experimental.ml import GeneformerTokenizer
+except ImportError:
+    # this should only occur when not running `experimental`-marked tests
+    pass
+
+
+def select_census_version_for_geneformer_tests() -> str:
+    # GeneformerTokenizer needs the "normalized" X layer which wasn't yet available in
+    # "stable" at the time this was written. This should provide a graceful transition
+    # once we next advance "stable" (after which it could be eliminated).
+    with cellxgene_census.open_soma(census_version="stable") as stable:
+        if "normalized" in stable["census_data"]["homo_sapiens"].ms["RNA"].X:
+            return "stable"
+    return "2023-09-04"
```

Comment:
This means that we should label this release to not be deleted anytime soon, otherwise the test will just fail. Can you coordinate with @atolopko-czi and @ebezzi  on this